distribution/assembly.xml

```diff
@@ -19,6 +19,11 @@
         <outputDirectory>lib</outputDirectory>
         <destName>source.war</destName>
       </file>
+      <file>
+        <source>${project.basedir}/../opengrok-tools/dist/opengrok-tools-1.1rc48.tar.gz</source>
```

Comment:
hard-coded version ?

Reply:
Also, will this mean that user will have to extract the tools separately after extracting the distribution tarball ?

Reply:
No, following the instalattion instruction, the tar.gz file is a python package

```bash
python3 -m pip install opengrok_tools-{version}.tar.gz
```

Reply:
So extracting the distribution tarball and installing tools with python from the tools subdirectory.

Reply:
I removed the hardcoded version but it will still break in the future.
The problem here is that the python setup.py removes the dash from the version name 1.1-rc48 making it only 1.1rc48.

Reply:
Or eventually if we have the domain opengrok.org we can have there self hosted pypi repository for the opengrok tools https://pypi.org/project/pypiserver/